docroot/themes/custom/twentyeighteen/templates/layout/page--404.html.twig

```diff
@@ -0,0 +1,130 @@
+<div {{ attributes.setAttribute('id', 'pg__c') }}>
+
+  <!-- ______________________ BRANDING _______________________ -->
+
+  <header id="branding">
+    <div class="cog--mq mq-branding">
+      {% if page.branding %}
+        <div id="branding-region">
+          {{ page.branding }}
+        </div>
+      {% endif %}
+    </div>
+  </header>
+  <!-- /branding -->
+
+  <!-- ______________________ HEADER _______________________ -->
+
+  <header id="header">
+    <div class="cog--mq mq-header">
+      {% if page.header %}
+        <div id="header-region">
+          {{ page.header }}
+        </div>
+      {% endif %}
+    </div>
+  </header>
+  <!-- /header -->
+
+  <!-- ______________________ Pre content _______________________ -->
+  {% if page.pre_content %}
+    <div class="clearfix pre-content" id="pre-content">
+      <div class="cog--mq mq-precontent">
+        {{ page.pre_content }}
+      </div>
+    </div>
+  {% endif %}
+
+  <!-- ______________________ MAIN _______________________ -->
+
+  <div class="clearfix" id="main">
+    <div class="cog--mq mq-main error-container notfound-backing">
+
+      {% if page.sidebar_first %}
+        <aside class="sidebar-first sidebars" id="sidebar-first">
+          {{ page.sidebar_first }}
+        </aside>
+      {% endif %}
+      <!-- /sidebar-first -->
+
+      <section id="content">
+
+        {% if breadcrumb or title or messages or tabs or action_links %}
+          <div id="content-header">
+
+            {{ breadcrumb }}
+
+            {% if page.highlighted %}
+              <div id="highlighted">{{ page.highlighted }}</div>
+            {% endif %}
+
+            {{ title_prefix }}
+
+            {% if title %}
+              <h1 class="title">{{ title }}</h1>
+            {% endif %}
+
+            {{ title_suffix }}
+            {{ messages }}
+            {{ page.help }}
+
+            {% if tabs %}
+              <div class="tabs">{{ tabs }}</div>
+            {% endif %}
+
+            {% if action_links %}
+              <ul class="action-links">{{ action_links }}</ul>
+            {% endif %}
+
+          </div>
+          <!-- /#content-header -->
+        {% endif %}
+
+        <div id="content-area" class="error-notice">
+          <h1>404-Not Found</h1>
+          <p>Can't find what you‘re looking for?</p>
+          <p>Try again, log in <a href="/user/login">here</a>, or contact the&nbsp;<a href="mailto:drupal4gov@gmail.com">site administrator</a>.</p>
```

Comment:
For accessibility reasons, we should not have the hyperlink be "here". Instead I would have `<a href="/user/login">log in here</a>`

Reply:
Done.